### PR TITLE
Account for observed & various models values being null

### PR DESCRIPTION
**Pull request by Kyubinhan**
* Rename some of the react components
* Create a route `/morecast` and redirect `/fire-weather` to the route (since this route has been already exposed to users)
* Update getInnerText type definitions in `utils/d3`
* Add more tests for bias-adjusted models

### CODE REVIEW

File: src/features/fireWeather/components/NoonForecastTable.tsx [hunk 1]
```diff
@@ -61,15 +61,15 @@ const NoonForecastDisplay = ({ values, testId, title }: Props) => {
                 <TableCell>Temp (°C)</TableCell>
                 {values.map(v => (
                   <TableCell key={v.datetime}>
-                    {v.temperature.toFixed(MODEL_VALUE_DECIMAL)}
+                    {v.temperature && v.temperature.toFixed(MODEL_VALUE_DECIMAL)}
```
Comment (andrea-williams) [inline src/features/fireWeather/components/NoonForecastTable.tsx:64]: what happens if v.temperature is null? Is the table cell just empty?

File: src/features/fireWeather/components/graphs/TempRHGraph.tsx [hunk 2]
```diff
@@ -542,34 +546,41 @@ const TempRHGraph: React.FunctionComponent<Props> = ({
         width,
         height,
         data: weatherValues,
+        textTestId: 'temp-rh-tooltip-text',
+        bgdTestId: 'temp-rh-graph-background',
         getInnerText: ([key, value]) => {
-          if (key === 'date') {
+          if (key === 'date' && typeof value === 'object') {
```
Comment (andrea-williams) [inline src/features/fireWeather/components/graphs/TempRHGraph.tsx:552]: what's the purpose of adding in `&& typeof value === 'object'` ?

File: src/features/fireWeather/components/NoonForecastTable.tsx (hunk 1 above)
Comment (Kyubinhan) [reply src/features/fireWeather/components/NoonForecastTable.tsx:64]: Yeap you can actually check this by running cypress tests. (Look at the first column of the observed value table)

File: src/features/fireWeather/components/graphs/TempRHGraph.tsx (hunk 2 above)
Comment (andrea-williams) [reply src/features/fireWeather/components/graphs/TempRHGraph.tsx:552]: nvm. I read 3 lines down and then it made sense

Comment (Kyubinhan) [reply src/features/fireWeather/components/graphs/TempRHGraph.tsx:552]: I am making sure that the value is the Date object.

**Review by andrea-williams — APPROVED** [review]

**Pull request merged**